<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>play_page</class>
 <widget class="QMainWindow" name="play_page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>黄金点游戏</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_round">
    <property name="geometry">
     <rect>
      <x>215</x>
      <y>10</y>
      <width>165</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(229, 229, 229);</string>
    </property>
    <property name="text">
     <string>回  合：1</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>481</width>
      <height>320</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/res/blank.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>320</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="MyButton" name="calculate">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>375</y>
      <width>180</width>
      <height>40</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>计  算  G  值</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>17</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(229, 229, 229);</string>
    </property>
    <property name="text">
     <string>上轮 G 值：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_G">
    <property name="geometry">
     <rect>
      <x>545</x>
      <y>120</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe Print</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(0, 85, 255);
background-color: rgb(0, 255, 127);</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="MyButton" name="btn_show">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>350</y>
      <width>115</width>
      <height>30</height>
     </rect>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>0</red>
          <green>170</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="ButtonText">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>120</red>
          <green>120</green>
          <blue>120</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>方正粗黑宋简体</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>得分榜</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/medal.png</normaloff>:/res/medal.png</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="error">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>380</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <bold>false</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string>有玩家未输入昵称或点数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_back">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>720</width>
      <height>450</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_back</zorder>
   <zorder>label_round</zorder>
   <zorder>scrollArea</zorder>
   <zorder>calculate</zorder>
   <zorder>label</zorder>
   <zorder>line_G</zorder>
   <zorder>btn_show</zorder>
   <zorder>error</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>720</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menustart">
    <property name="title">
     <string>start</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/home.png</normaloff>:/res/home.png</iconset>
    </property>
    <addaction name="action_back"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menuhelp">
    <property name="title">
     <string>help</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/help.png</normaloff>:/res/help.png</iconset>
    </property>
    <addaction name="action_rule"/>
   </widget>
   <widget class="QMenu" name="menulook">
    <property name="title">
     <string>menue</string>
    </property>
    <property name="icon">
     <iconset resource="res.qrc">
      <normaloff>:/res/look.png</normaloff>:/res/look.png</iconset>
    </property>
    <addaction name="action_score"/>
    <addaction name="action_history"/>
    <addaction name="action_trend"/>
   </widget>
   <addaction name="menustart"/>
   <addaction name="menuhelp"/>
   <addaction name="menulook"/>
  </widget>
  <action name="action_back">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/return.png</normaloff>:/res/return.png</iconset>
   </property>
   <property name="text">
    <string>返回上页</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/quit.png</normaloff>:/res/quit.png</iconset>
   </property>
   <property name="text">
    <string>退出游戏</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F4</string>
   </property>
  </action>
  <action name="action_rule">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/rule.png</normaloff>:/res/rule.png</iconset>
   </property>
   <property name="text">
    <string>游戏规则</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="action_score">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/medal.png</normaloff>:/res/medal.png</iconset>
   </property>
   <property name="text">
    <string>得分榜</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_history">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/history.png</normaloff>:/res/history.png</iconset>
   </property>
   <property name="text">
    <string>历史记录</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_trend">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/trend.png</normaloff>:/res/trend.png</iconset>
   </property>
   <property name="text">
    <string>G值走势</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyButton</class>
   <extends>QPushButton</extends>
   <header>mybutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
